aws/resource_aws_autoscaling_policy_test.go

```diff
@@ -375,3 +415,79 @@ resource "aws_autoscaling_policy" "foobar_simple" {
 }
 `, name, name, name)
 }
+
+func testAccAwsAutoscalingPolicyConfig_TargetTracking_Predefined(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_launch_configuration" "test" {
+  name          = "tf-test-%s"
+  image_id      = "ami-21f78e11"
+  instance_type = "t1.micro"
+}
+
+resource "aws_autoscaling_group" "test" {
+  availability_zones        = ["us-west-2a"]
```

Comment:
Do you mind using the AZ data source here to make the test region-agnostic?
https://www.terraform.io/docs/providers/aws/d/availability_zones.html